### PR TITLE
copy events from existing instance

### PR DESCRIPTION
**Pull request by vetinari**
otherwise we produce flapping alerts like

```
2017-11-02 04:08:01     ALERT_ENTITY_STARTED
"entity-name-H4Sh01[aws:1234567890:eu-central-1]"
{"td":3.89E-4,"worker":"plocal.ip-172-31-132-123","ts":1.50959207725566E9,"value":[{"Code":"instance-stop","Description":"The instance is running on degraded
hardware","NotBefore":"2017-11-14T22:00:00+00:00"}]}
2017-11-02 04:18:00     ALERT_ENTITY_ENDED
"entity-name-H4Sh01[aws:1234567890:eu-central-1]"
{"td":6.02E-4,"worker":"plocal.ip-172-31-132-123","ts":1.509592677890665E9,"value":[]}
2017-11-02 08:06:59     ALERT_ENTITY_STARTED
"entity-name-H4Sh01[aws:1234567890:eu-central-1]"
{"td":4.84E-4,"worker":"plocal.ip-172-31-140-110","ts":1.509606418656158E9,"value":[{"Code":"instance-stop","Description":"The instance is running on degraded
hardware","NotBefore":"2017-11-14T22:00:00+00:00"}]}
2017-11-02 08:16:59     ALERT_ENTITY_ENDED
"entity-name-H4Sh01[aws:1234567890:eu-central-1]"
{"td":5.74E-4,"worker":"plocal.ip-172-31-140-110","ts":1.50960701801336E9,"value":[]}
```

### CODE REVIEW

**Comment by vetinari** [comment]
:+1:

File: zmon_aws_agent/aws.py [hunk 1]
```diff
@@ -295,9 +295,14 @@ def get_running_apps(region, existing_entities=None):
...
                         ins['name'] = tags['Name'].replace(' ', '-')
 
-            if 'application_id' in ins and not (now.minute % 10):
-                ins['events'] = get_instance_events(aws_client, i)
-                ins['block_devices'] = get_instance_devices(aws_client, i)
+            if 'application_id' in ins:
+                if not (now.minute % 10):
+                    ins['events'] = get_instance_events(aws_client, i)
+                    ins['block_devices'] = get_instance_devices(aws_client, i)
+                else:
```
Comment (a1exsh) [inline zmon_aws_agent/aws.py]: What about `block_devices`?  Shouldn't we copy those as well?

Comment (vetinari) [reply zmon_aws_agent/aws.py]: They're filled (line 244) and not cleared explicitly like the events (line 255)

Comment (a1exsh) [reply zmon_aws_agent/aws.py]: Doesn't that mean we get block devices two times per instance if it happens to run on a round 10-minute number?

**Comment by a1exsh** [comment]
:+1:

File: zmon_aws_agent/aws.py (hunk 1 above)
Comment (a1exsh) [reply zmon_aws_agent/aws.py]: OK, I think this can be fixed separately.

**Comment by vetinari** [comment]
:+1:

**Comment by vibhory2j** [comment]
👍

**Comment by vetinari** [comment]
:+1:

**Pull request merged**